Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -4622,6 +4635,11 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"\"$(PLATFORM_DIR)/Developer/Library/Frameworks\"",
```

Comment:
ohhhhhhhheeeeeerrrrrrrrrrrrrrmmmmmmmmmmhmm I did add that second line when the internet told me to because something called XCTest.h was not found when building. But I later discovered that the problem was that my test file was somehow added to the library, and not just to the tests. So these lines probabably don't need to change here after all.